packages/lesswrong/server/sideComments.ts

```diff
@@ -4,7 +4,11 @@ import { Comments } from '../lib/collections/comments/collection';
 import type { SideCommentsResolverResult } from '../lib/collections/posts/schema';
 import { getDefaultViewSelector } from '../lib/utils/viewUtils';
 import groupBy from 'lodash/groupBy';
-import some from 'lodash/some';
+import { getToCforPost } from './tableOfContents';
+import { createMutator } from './vulcan-lib/mutators';
+import { SideCommentsCache } from "../lib/collections/sideCommentsCache/collection";
+
+export const sideCommentCacheVersion = 1;
```

Comment:
This is something like a "format/syntax of side comments" version?

Reply:
Right. If we bump this version number, all cache entries with the old version number are treated as invalid.